packages/broker/src/ConfigWizard.ts

```diff
@@ -250,14 +252,19 @@ export async function startBrokerConfigWizard(): Promise<void> {
     try {
         const answers = await inquirer.prompt(prompts)
         const config = getConfigFromAnswers(answers)
-        logger.info(`This will be your node's address: ${new Wallet(config.ethereumPrivateKey).address}`)
+        const nodeAddress = new Wallet(config.ethereumPrivateKey).address
+        const mnemonic = Protocol.generateMnemonicFromAddress(nodeAddress)
+        logger.info('Welcome to the Streamr Network')
+        logger.print(`Your node's generated name is ${mnemonic}.`)
+        logger.print('View your node in the Network Explorer:')
+        logger.print(`https://streamr.network/network-explorer/nodes/${nodeAddress}`)
```

Comment:
Fyi, Network explorer expects the full node id, not address... though this form is more user friendly